api-client/src/test/java/com/xing/api/CallSpecTest.java

```diff
@@ -399,6 +399,48 @@ public void builderAllowsJsonBody() throws Exception {
         assertRequestHasBody(callSpec.request(), expectedBody, 24);
     }
 
+    @Test
```

Comment:
I'm unsure about if we need these tests, the logic they are testing, is already tested right? Do we need to re-test it again for the different methods? GET, POST, PATCH?

Reply:
We don't need to specifically check for each HTTP method, but all the others are being tested, so I added tests for PATCH too. I can remove them.